.circleci/config.yml

```diff
@@ -260,23 +268,25 @@ jobs:
           command: |
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
               . activate habitat; cd habitat-lab
-              python setup.py develop --all
-              find . -name ci-requirements.txt | xargs -I {} pip install -r {}
+              pip install -e habitat-lab
+              pip install -e habitat-baselines
               export PYTHONPATH=.:$PYTHONPATH
               export MULTI_PROC_OFFSET=0 && export MAGNUM_LOG=quiet && export HABITAT_SIM_LOG=quiet
-              python setup.py test --addopts " --cov-report=xml --cov-report term  --cov=./"
+              python -m pytest --cov-report=xml --cov-report term  --cov=./
```

Comment:
I removed this option since it seems that `python setup.py test` is deprecated. https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#do-not-run-via-setuptools